core/color.cpp

```diff
@@ -327,7 +329,24 @@ bool Color::html_is_valid(const String& p_color) {
 
 }
 
-
+Color Color::named(const String &p_name) {
+	if(_named_colors.empty()) _populate_named_colors(); // from color_names.inc
```

Comment:
Any chance to use space after if? I hope in 3.0 there will be some codestyle enforced. In other files you're using this space after if...